packages/tideline/plugins/blip/chartdailyfactory.js

```diff
@@ -89,70 +90,91 @@ function chartDailyFactory(parentElement, tidelineData, options = {}) {
 
   // top x-axis pool
   /** @type {Pool} */
-  const poolXAxis = chart.newPool()
-    .id('poolXAxis', chart.poolGroup)
-    .label('')
-    .labelBaseline(options.labelBaseline);
-  poolXAxis.index(chart.pools.indexOf(poolXAxis))
+  const poolXAxis = new Pool(chart);
+  chart.addPool(poolXAxis);
```

Comment:
the way we create those objects are really strange.
I create a pool using the chart container
```
  const poolXAxis = new Pool(chart);
```
and then I add this new created pool to the list of pools of the original object
```
  chart.addPool(poolXAxis);
```

Reply:
Yes I know.
I change all theses little things steps by steps.
Someday I hope ....